src/access/accesscontrol/accesscontrol.cairo

```diff
@@ -138,7 +137,7 @@ mod AccessControl {
     #[generate_trait]
     impl InternalImpl of InternalTrait {
         fn initializer(ref self: ContractState) {
-            let mut unsafe_state = SRC5::unsafe_new_contract_state();
+            let mut unsafe_state = src5_state();
```

Comment:
why not inlining in the next line like the diff above?

Reply:
to pass `ref`s down, we need to move it into a variable first, in this example that can't be inlined (compilation error).